src/prefect/runner/server.py

```diff
@@ -80,6 +95,8 @@ async def _create_flow_run_for_deployment(
             flow_run = await client.create_flow_run_from_deployment(
                 deployment_id=deployment.id, parameters=body
             )
+        runner.execute_in_background(runner.execute_flow_run, flow_run.id)
```

Comment:
Oh nice! I see you're experimenting with cutting out the API call! Excited to see how that's working!